### DISCUSSION TITLE
Going to 'resize' mode by 'Alt+R' and can't escape it

### DISCUSSION THREAD

**Discussion by pressslav** [glzr-io/glazewm#785] [Q&A] [answered]
Hey, is there something I'm missing? - Every time I go into resizing mode I simply cannot go back , is there some shortcut I'm not seeing or? It literally stays in resizing mode (as indicated in the zebar above and most of my keyboard keys are inactive and the only way to exit is to restart glazewm?

Any help would be appreciated as I enjoy the product, it's what keeping me from moving fully over to Linux ;/

**Comment by lars-berger** [comment, marked as answer]
Enter or escape key. You can double check what it's bound to in the user config

**Reply by pressslav** [reply]
Yep, that works, thank you. Apologies for the ignorance!

**Reply by lars-berger** [reply]
No worries at all 👌
